index.html

```diff
@@ -6,5 +6,54 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Personal website</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>Borislav Borislavov</h1>
+      <section class="topnav">
+        <a href="#contact">Contact</a>
+        <a href="#bio">Biography</a>
+      </section>
+    </header>
+    <br />
+    <main>
+      <img
+        src="https://profile-images.xing.com/images/18f0cdb39a7fa0b8966ef3084f29ad92-1/borislav-borislavov.256x256.jpg"
+        alt="Profile picture from xing.com"
+      />
+      <br>
+      <h2>Interests</h2>
+      <ul>
+          <li>Web development</li>
+          <li>Reading sci fi books</li>
+          <li>Beekeeping</li>
+      </ul>
+      <br>
+      <h2>Skills</h2>
+      <ul>
+          <li>a</li>
+          <li>b</li>
+          <li>c</li>
+      </ul>
+      <br>
+      <section>
+          <!--Add a border to this section-->
```

Comment:
You should remove this comment. I think this is just a mental note, but you don't want this in your production code.

Reply:
you are right, comment removed.